File: Java/Estrutura condicional/Multiplos.java
import java.util.Locale;
import java.util.Scanner;

public class Multiplos {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int a, b;

        System.out.println("Digite dois numeros inteiros: ");
        a = sc.nextInt();
        b = sc.nextInt();

        if (a % b == 0 || b % a == 0) {
            System.out.println("Sao multiplos");
        } else {
            System.out.println("Nao sao multiplos");
        }

        sc.close();

    }
}

File: C++/Estrutura condicional/Multiplos.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int n1, n2;

    cout << "Digite dois numeros inteiros: " << endl;
    cin >> n1;
    cin >> n2;

    if (n1 % n2 == 0 || n2 % n1 == 0)
    {
        cout << "Sao multiplos";
    }
    else
    {
        cout << "Nao sao multiplos";
    }
    return 0;
}